Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+def docker_images = ["librecores/hosting", " librecores/librecores-ci-modules-fpga", "librecores/ci-modules" , "librecores/librecores-ci-riscv", "librecores/librecores-ci"]
+
+def get_stages(docker_image) {
+    stages = {
+        docker.image(docker_image).inside {
+            stage("${docker_image}") {
+                echo 'Running in ${docker_image}'
+            }
+            stage("Build") {
+                sh 'echo this is stage Build'
+                container = docker.build('docker_image')
+            }
+            stage("Test") {
+                sh 'echo this is stage Test'
+                sh 'export GOSS_FILES_STRATEGY=cp && /usr/local/bin/dgoss  run --name jenkins-docker-dgoss-test 
+                --rm -ti docker_image'
+            }
+        }
+    }
+    return stages
+}
+
+node('master') {
```

Comment:
would be nice to enable a retention strategy here. 
E.g. https://github.com/jenkinsci/jenkins/blob/master/Jenkinsfile#L16

Reply:
Not sure whether it works inside `node()`, but I will give it a try